authors/settings.py

```diff
@@ -81,8 +81,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.environ.get('DB_NAME', 'authors'),
+        'USER': os.environ.get('DB_USER', ''),
+        'PASSWORD': os.environ.get('DB_PASS', ''),
+        'HOST': 'localhost',
```

Comment:
@Emmanuelaaron I notice you have set '**HOST**' and '**PORT**' statically, I suggest you set them to pick their values from the environment variables just as you have done for the rest to avoid future errors when deployment on online platforms.

Reply:
I see you have resolved. great work!!!